mxabierto/adela
# Create task fechas

## Changed:

### M : The next files has been modified:

*   db/migrate/20170712054823_split_temporal_data.rb
*   db/schema.rb
* lib/tasks/create_published_date.rake

### How to test (Steps)
1) Download the source in your local environment.
2) execute rake db:migrate RAILS_ENV=development
3) In your rails console execute Dataset.last
4) You must see the fields temporal_init_date and temporal_term_date in the Dataset
5) Execute Distribution.last 
6) You must see the fields temporal_init_date and temporal_term_date in the Distribution
